frontend/src/components/explorer/gemBrowser/MapsAvailable.vue

```diff
@@ -6,34 +6,39 @@
         <span>{{ messages.mapViewerName }}</span>
       </p>
     </header>
-    <div v-if="mapAvailableLimited" class="card-content p-2">
-      <div v-for="mapKey in mapKeys" :key="mapKey"
-           class="content has-text-left p-0">
-        <template>
-          <div>{{ mapKey.toUpperCase() }} maps</div>
-          <ul class="py-0 px-4">
-            <template v-for="mapType in Object.keys(mapAvailableLimited[mapKey])">
-              <template v-for="map in mapAvailableLimited[mapKey][mapType]">
-                <li :key="map.id">
-                  <router-link v-if="viewerSelectedID"
-                               :to="{ name: 'viewer',
-                                      params: { model: model.short_name, type: mapType, map_id: map.id, reload: true },
-                                      query: { dim: mapKey, search: viewerSelectedID, sel: viewerSelectedID } }">
+    <div v-if="mapsAvailable.length !== 0" class="card-content p-2">
+      <table class="table test-table">
+        <tbody class="has-text-left">
+          <tr v-for="component in mapsAvailable" :key="component.id">
+            <td> {{ component.customName }} </td>
```

Comment:
Since the maps available component is supposed to go in a column that `is-narrow`, which would have a flexible width, it might be good to limit the width of the component name. Not sure if there is a built-in Bulma helper for max-width but you can always access them with a CSS selector, for example: `.test-table tr td:first-child {`.